### ISSUE TITLE
Incorrect validation color

### ISSUE THREAD

**Issue by zhaolinlau** [oruga-ui/theme-bulma#99]
### Overview of the problem

**Oruga** version: [0.8.5]
**Vuejs** version: [3.4.21]
**OS/Browser**: Windows 11, Firefox

### Description

When i submit a form without filling any value, the inputs should be turned to red color and red color message will be shown. However, the color is incorrect.

### Steps to reproduce

1. add required attribute to `<OInput />`

### Expected behavior

![image](https://github.com/oruga-ui/theme-bulma/assets/68846168/ba7f9c91-5a6d-46b5-a456-0a123983b9f6)

### Actual behavior

![image](https://github.com/oruga-ui/theme-bulma/assets/68846168/b8f9dc15-2c64-4e65-a95d-d726836c8c07)

**Comment by blm768** [comment]
I've confirmed this behavior in my own environment as well. Here's a minimal example component that demonstrates the issue:

```vue
<script setup lang="ts">
import { OField, OInput } from '@oruga-ui/oruga-next';

import { ref } from 'vue';

const inputValue = ref("");
</script>

<template>
  <main>
    <form>
      <OField label="Input">
        <OInput v-model="inputValue" required />
      </OField>
    </form>
  </main>
</template>
```

It seems that this issue isn't strictly specific to the Bulma theme, though; I'm also observing it with the Oruga theme. It seems that the prefix of the variant class (e.g. `o-input--` or `is-`) is getting applied to the input, but the suffix (e.g. `danger`) is left off.

**Comment by blm768** [comment]
[This commit](https://github.com/oruga-ui/oruga/commit/ed75a718d3282198e532596423ea4b71256cfb06) seems to have fixed the issue for Oruga 0.8.6.
